app/src/app/client.cljs

```diff
@@ -227,18 +238,34 @@
 
 (defn page []
   [:div
-   [:div {:class "header"
-          :style {:display         "flex"
-                  :flex-direction  "row"
-                  :justify-content "space-around"}}
-    ;; layout hack: empty divs move the title and sign-off button more to the center
-    [:div]
-    [:div {:style {:font-size "1.3em"}} "ccDashboard"]
-    (if (domain/user-sign-in-state @domain/app-state)
-      [:a#logout {:href "/logout"}
-       [:i.icon-off.large-icon]]
-      [:div])
-    [:div]]
+   ;[:div {:class "header"
+   ;       :style {:display         "flex"
+   ;               :flex-direction  "row"
+   ;               :justify-content "space-around"}}
+   ; ;; layout hack: empty divs move the title and sign-off button more to the center
+   ; [:div]
+   ; [:a {:href  "/#"
+   ;      :style {:font-size "1.3em"}}
+   ;  "ccDashboard"]
+   ; (sign-in-only-content [:a#location {:href "/#location"
+   ;                                     :style {:font-size "1.3em"}}
+   ;                        "Location View"])
+   ; (sign-in-only-content [:a#logout {:href "/logout"}
+   ;                        [:i.icon-off.large-icon]])
+   ; [:div]]
+   [:div#navbar
+    [:label.show-menu {:for "show-menu"}
+     [:i.icon-menu.large-icon]]
+    [:span#banner
+     [:a {:href "/#"} "ccDashboard"]]
+    [:input#show-menu {:type "checkbox"
+                       :role "button"}]
+    [:span#menu
+     [:ul
+      (map (fn [[h s]] ^{:key h} [:li [:a {:href h} s]])
```

Comment:
`h` and `s` is short, but not very descriptive. Better go with longer names in general, except for "well-known" names like `k` & `v`